snuba_sdk/visitors.py

```diff
@@ -228,3 +228,82 @@ def _visit_turbo(self, turbo: Turbo) -> str:
 
     def _visit_debug(self, debug: Debug) -> str:
         return str(debug)
+
+
+class ExpressionFinder(ExpressionVisitor[Set[Expression]]):
+    def __init__(self, exp_type: Any) -> None:
```

Comment:
Can this type be restricted or would this be a massive Union ?

Reply:
It would be massive because it's the type of any Expression.